README.md

```diff
@@ -228,253 +376,406 @@ Your directory structure should look like this:
+database. To send this request, we'll enter the request URL:
+"http://localhost:8080/football-team/1". Remember to select "DELETE" next to the
+request URL before clicking "Send".
 
-```java
-@Override
-public void run(String... args) {
-	Optional<Teacher> teacher1Optional = repository.findById(1);
-	Teacher teacher1 = teacher1Optional.orElseThrow(() -> new Error("object not found"));
-	repository.delete(teacher1);
-}
-```
+![postman-delete-football-team](https://curriculum-content.s3.amazonaws.com/spring-mod-1/repository/postman-delete-football-team.png)
+
+## Relationships in Spring Data JPA
+
+Since Spring Data JPA is a complete implementation of the JPA specification,
+its relationship annotations are the same. We can use all the annotations we
+learned in the JPA and Hibernate section such as `@OneToOne`, `@OneToMany`, and
+`@ManyToMany`. These annotations can still be used to define relationships
+between entities within a Spring Boot application. The repositories will be able
+to fetch the required data with relationships.
+
+## Custom Queries
+
+As we saw before, the query builder is pretty handy! The derived queries make it
+easy for us; however, the method name can get pretty messy. For example,
+consider the derived query method name:
 
 ```java
-Hibernate: select teacher0_.id as id1_0_0_, teacher0_.name as name2_0_0_ from teacher teacher0_ where teacher0_.id=?
-Hibernate: select teacher0_.id as id1_0_0_, teacher0_.name as name2_0_0_ from teacher teacher0_ where teacher0_.id=?
-Hibernate: delete from teacher where id=?
+    Optional<FootballTeam> findTopByWinsAfter(Integer lowerBound);
 ```
 
-The first record in the database table should now be deleted:
+We might be thinking "What does that method even do?" and that is because it may
+not seem completely obvious or intuitive. The method above equates to the SQL
+query:
 
-| ID  | NAME |
-| --- | ---- |
-| 2   | Ian  |
+```postgresql
+SELECT *
+FROM football_team
+WHERE wins > lowerBound
+LIMIT 1;
+```
 
-## Relationships in Spring Data JPA
+The query itself, in this case, is a little easier to understand. When the query
+we are interested in gets a little too complex, it might be beneficial to use a
+declared custom query with the `@Query` annotation. We could rewrite this method
+to be:
 
-Since Spring Data JPA is a feature complete implementation of the JPA
-specification, its relationship annotations are the same. We can use all of the
-annotations we learned in the JPA and Hibernate section such as `@OneToOne`,
-`OneToMany`, and `ManyToMany` to define relationships between entities. The
-repositories will be able to fetch the required data with relationships.
+```java
+    @Query(value = "SELECT * FROM football_team WHERE wins > ?1 LIMIT 1", nativeQuery = true)
+    Optional<FootballTeam> findTeamWithWinsMoreThan(Integer lowerBound);
+```
+
+This is a more hands-on approach when it comes to writing queries and allows us to
```

Comment:
Consider mentioning that native queries make it harder to switch to a different database provider.
At my last job, we were trying to transition from Oracle to SQL Server, so we had to figure out how to rewrite a hundred native queries.

Reply:
Mentioned!